psi4/driver/procrouting/scf_proc/subclass_methods.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+from psi4 import core
+
+def _RHF_orbital_gradient(self, save_fock: bool, max_diis_vectors: int):
```

Comment:
Can you add return annotations as well?

```python
def _RHF_orbital_gradient(self, save_fock: bool, max_diis_vectors: int) -> float:
```

Reply:
Sure, I'll include that in the next set of commits.

Normally when somebody who isn't a core developer leaves comments on a PR, the PR is directly relevant to their research. Can I ask if there's some special reason for you commenting on this one?

Reply:
@JonathonMisiewicz not particularly. I'm using some of the python SCF API lately, glad to see more logic available at the python level. In general type hinting FTW!

Reply:
@loriab Someone ought to note that in the "developers howto", if it's not there already.